lib/chef/resource/windows_task.rb

```diff
@@ -59,6 +59,8 @@ class WindowsTask < Chef::Resource
       property :execution_time_limit, [String, Integer], default: "PT72H" # 72 hours in ISO8601 duration format
       property :minutes_duration, [String, Integer]
       property :minutes_interval, [String, Integer]
+      property :disallow_start_on_battery, [TrueClass, FalseClass], default: false
```

Comment:
We'll want to add description and introduced fields to all of these properties. We introduced the description and introduced in Chef 14.0 and we're going to be using it in the future to generate documentation.

https://github.com/chef/chef/pull/7475/files#diff-b03a778f5af81b89ff2db80b67f7adc0R100 is a good example of how we use both of them.

Reply:
Thank you @tas50 for nice suggestion. I will add fields `description` and `introduced` for the properties.